src/main/java/org/example/servletstexts/service/TextService.java

```diff
@@ -0,0 +1,36 @@
+package org.example.servletstexts.service;
+
+
+import org.example.servletstexts.dao.TextsDAO;
+
+
+public class TextService {
+
+    private final TextsDAO textsDAO;
+
+    public TextService(TextsDAO textsDAO) {
+        this.textsDAO = textsDAO;
+    }
+
+    public int addText(String text) {
+        return textsDAO.add(text);
+    }
+
+    public String getTextById(int id) {
+        return textsDAO.get(id);
+    }
+
+    public String getAll() {
+        return textsDAO.getAll().toString();
```

Comment:
Use case : 
I'm as a user of the service (TextService). Another developer who develop the text service. 
I've got requirement to respond with all texts which has length < 1000 symbols.  This service looking very useful for me and I would like to reuse the functionality. But I've faced with issue that your textService return all texts as String. So, how can I filter texts in order to satisfy my requirements ?